tests/testTriggers.go

```diff
@@ -6908,6 +6910,157 @@ func TriggerConfluentAsyncDR(contexts *[]*scheduler.Context, recordChan *chan *E
 	})
 }
 
+func TriggerKafkaAsyncDR(contexts *[]*scheduler.Context, recordChan *chan *EventRecord) {
+	defer endLongevityTest()
+	startLongevityTest(KafkaAsyncDR)
+	defer ginkgo.GinkgoRecover()
+	log.InfoD("Kafka CRD Async DR triggered at: %v", time.Now())
+	event := &EventRecord{
+		Event: Event{
+			ID:   GenerateUUID(),
+			Type: KafkaAsyncDR,
+		},
+		Start:   time.Now().Format(time.RFC1123),
+		Outcome: []error{},
+	}
+	defer func() {
+		event.End = time.Now().Format(time.RFC1123)
+		*recordChan <- event
+	}()
+
+	setMetrics(*event)
+
+	chaosLevel := ChaosMap[KafkaAsyncDR]
+	var (
+		includeVolumesFlag    = true
+		includeResourcesFlag  = true
+		startApplicationsFlag = true
+		ns_name               = "kafka"
+		repoName              = "strimzi"
+		operatorName          = "strimzi-kafka-operator"
+		migrationList         []*storkapi.Migration
+		app_url               = "/root/torpedo/deployments/customconfigs/kafkacr.yaml"
+	)
+
+	Step(fmt.Sprint("Export kubeconfigs"), func() {
+
+		// Write kubeconfig files after reading from the config maps created by torpedo deploy script
+		err := asyncdr.WriteKubeconfigToFiles()
+		if err != nil {
+			log.Errorf("Failed to write kubeconfig: %v", err)
+			return
+		}
+
+		err = SetSourceKubeConfig()
+		if err != nil {
+			log.Errorf("Failed to Set source kubeconfig: %v", err)
+			return
+		}
+	})
+	Step(fmt.Sprintf("Deploy applications with %v chaos level", chaosLevel), func() {
+		for i := 0; i < Inst().GlobalScaleFactor; i++ {
+			//taskName := fmt.Sprintf("%s-%d", taskNamePrefix, i)
+			ns, err := core.Instance().GetNamespace(ns_name)
+			if err != nil {
+				log.InfoD("Creating namespace %v", ns_name)
+				nsSpec := &v1.Namespace{
+					ObjectMeta: meta_v1.ObjectMeta{
+						Name: ns_name,
+					},
+				}
+				ns, err = core.Instance().CreateNamespace(nsSpec)
+				if err != nil {
+					UpdateOutcome(event, fmt.Errorf("Failed to create namespace, err: %v", err))
+					return
+				}
+
+			}
+			log.InfoD("Pods Creation Started")
+			pods_created, err := asyncdr.HelmRepoAddandCrInstall(repoName, "https://strimzi.io/charts/", ns.Name, operatorName, fmt.Sprintf("%v/%v", repoName, operatorName), app_url)
+			if err != nil {
+				UpdateOutcome(event, fmt.Errorf("Failed to create app pods, err: %v", err))
+				return
+			}
+			pods_created_len := len(pods_created.Items)
+			log.InfoD("Num of Pods on source: %v", pods_created_len)
+			expected_kafka_crd_list := []string{"kafkabridges.kafka.strimzi.io", "kafkaconnectors.kafka.strimzi.io", "kafkaconnects.kafka.strimzi.io",
```

Comment:
Let's move this list `[]string{....}` into async DR package as a global variable, that way it will be easier to change, if required, in the future.

Reply:
Done!